slices/coreutils.yaml

```diff
@@ -129,6 +129,12 @@ slices:
       /usr/bin/yes:
       /usr/sbin/chroot:
 
+  env:
```

Comment:
since this slice is being introduced and it's a binary, can you please add a test for it?